packages/bcer-data-portal/app/src/views/Locations.tsx

```diff
@@ -132,25 +131,111 @@ const useStyles = makeStyles({
   },
 });
 
+const tableRef = React.createRef();
```

Comment:
What's this tableRef used for? i see it assigned to the table down below, but no further usage

Reply:
Good catch. It didnt end up being used